src/main/resources/assets/forge/lang/en_us.json

```diff
@@ -123,9 +123,9 @@
   "forge.configgui.removeErroringTileEntities": "Remove Erroring Tile Entities",
   "forge.configgui.fullBoundingBoxLadders.tooltip": "Set this to true to check the entire entity's collision bounding box for ladders instead of just the block they are in. Causes noticeable differences in mechanics, so the default is vanilla behavior. Default: false.",
   "forge.configgui.fullBoundingBoxLadders": "Full Bounding Box Ladders",
-  "forge.configgui.zombieBaseSummonChance.tooltip": "Base zombie summoning spawn chance. Allows changing the bonus zombie summoning mechanic.",
```

Comment:
Hey, word "zombie" doesn't have to be uppercase: https://minecraft.gamepedia.com/Zombie :)